README.md

```diff
@@ -241,20 +241,51 @@ friendly** by using hashes.
 
-If you are still having difficulty after looking over your configuration carefully, please post
-a question to [StackOverflow with the webpack tag](https://stackoverflow.com/tags/webpack). Questions
-that include your webpack.config.js and relevant files are more likely to receive responses.
+Contributions go far beyond pull requests and commits. Although we love giving you the opportunity to put your stamp on webpack, we also are thrilled to receive a variety of other contributions including:
 
-If you have discovered a bug or have a feature suggestion, feel free to create an issue on Github.
+* [Documentation](https://github.com/webpack/webpack.js.org) updates, enhancements, designs, or bugfixes
 
-If you create a loader or plugin, please consider open sourcing it, putting it
-on npm and following the `x-loader`, `x-webpack-plugin` convention.
+* Spelling or grammar fixes
 
-You are also welcome to correct any spelling mistakes or any language issues.
+* README.md corrections or redesigns
 
-If you want to discuss something or just need help, [here is our Gitter room](https://gitter.im/webpack/webpack).
+* Adding unit, or functional tests
+
+* Triaging GitHub issues (especially helping find out if an issue still persists or is reproducible)
+
+* [Searching #webpack on twitter](https://twitter.com/search?q=webpack) and helping someone else who needs help
+
+* Teaching others how to contribute to one of the many webpack repo's!
+
+* [Blogging, speaking about, or creating tutorials](https://github.com/webpack-contrib/awesome-webpack) about one of webpack's many features. 
+
+* Helping others in our webpack [gitter channel](https://gitter.im/webpack/webpack).
+
+If you are worried or don't know where to start, you can **always** reach out to [Sean Larkin (@TheLarkInn) on Twitter](https://twitter.com/thelarkinn) or simply submit an issue and a maintainer can help give you guidance! 
+
+We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributors Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help. 
+
+_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!! 
+
+<h3 align="center">Creating your own plugins and loaders</h3>
+If you create a loader or plugin, we would <3 for you to open source it, and put it on npm. We follow the `x-loader`, `x-webpack-plugin` naming convention.
+
+<h2 align="center">Support</h2>
+
+We consider webpack to be a low-level tool that is used by not only individuals but also layered beneath lots of other awesome tools. Because of this, we know that sometimes webpack is not the _easiest_ to configure, and we constantly are working on finding ways to improve this!!!
```

Comment:
I would rephrase slightly:

``` md
We consider webpack to be a low-level tool used not only individuals but also layered beneath other awesome tools. Because of it's flexibility, webpack isn't always the _easiest_ entry-level solution, however we do believe it is the most powerful. That said, we're always looking for ways improve and simplify the tool without compromising functionality. If you have any ideas on ways to accomplish this, we're all ears!
```